### DISCUSSION TITLE
Identity potions as a better way to trigger identities

### DISCUSSION THREAD

**Discussion by morceaudebois** [Draylar/identity#595] [Ideas]
Hi!

I absolutely love the Identity mod but I feel like the way you trigger the morphing is a little bit underwhelming. I thought of something that would make it more gameplay oriented and more rewarding.

You know how Valheim mobs drop trophies from time to time? It would be nice to integrate this into Minecraft (it already exists with mods like [Trophers](https://modrinth.com/mod/trofers)). With a mob's trophy, you would be able to brew a potion of identity that would give you the mob's identity. This effect would last something like 4 in-game days by default and then it would revert back to normal.

I would love to make it myself but I'm really new to modding, so I have no idea what I'm doing. I would probably be able to make the potions but I don't know if it's possible to interact with functions of another mod (mainly to trigger the identity when drinking the potion and use Trofers' trophies as ingredient). From what I read, [the API](https://github.com/Draylar/identity/wiki/Identity-API) doesn't seem to be able to do it so I'm not sure.

What do you guys think?

**Comment by CreeprVictor** [comment]
And mabey, for here, the starter potion would be the thick potion or something, as opposed to all the normal ones using awkward!
